cogs/git_monitor/git_monitor.py

```diff
@@ -60,11 +69,11 @@ def save_config(self):
 
     # Background commit checking loop which interacts with the GitHub API. 
     # The loop runs the check_guild_repos coroutine every 10 minutes, which in turn calls the check_repo_commits coroutine.
-    @tasks.loop(minutes=10.0)
+    @tasks.loop(minutes=5)
```

Comment:
Let bump this down. Doesnt need to be as aggressive. 30? 60? This is an async style fyi and shouldn't be the first notification if we need immediate attention.

Also update the comment above it to whatever the value should be.

Reply:
Don't see a problem with this. Also included a command to change the loop interval, but I will set it to 30 which will be default when the cog is loaded.